### ISSUE TITLE
Active cases column

### ISSUE THREAD

**Issue by ulklc** [ulklc/covid19-timeseries#7]
Current structure already have confirmed, death and cured. I am thinking to add active column with this formula (confirmed - (death + cured) = active).
`day | countryCode | countryName | region | lat | lon | confirmed | cured | death | *active*`
What do you think ? Should we add that ?

**Comment by chrisjbillington** [comment]
I would say no, but I don't feel too strongly about it. It's redundant and easily calculated by the user, and its presence could be misleading - the user might think "If this is here, does that mean it is something different to confirmed - recovered - deaths?"

While we're at it, 'cured' is an unusual label for the recoveries column. I am guessing you are not a native English speaker, so to explain, 'cured' has connotations of the recovery being due to treatment, and is not really used for when patients recover spontaneously. So 'recovered' would be the more natural word.

**Comment by ulklc** [comment]
Yes, i am not a native eng speaker, and i agree. "recovered" label instead of "cured" making more sense. But it is requiring some code changes in crawler. Actually I can change it before more user start to use this repo.

**Comment by ulklc** [comment]
#8

**Comment by ivanMSC** [comment]
No to `active` column. Including it in the repo will only add unnecessary info, and my spreadsheet can  only take so many bytes 😂 
In fact, if you ask me, I would get rid of columns `region`, `lat` and `lon`, and put them on a separate table for reference.

**Comment by iandoug** [comment]
I'm native English, I also use "cured" simply because it's shorter than "recovered".
There is a point of view that it is the doctor's job to entertain the patient while nature does the healing, so the difference may be somewhat artificial.... especially in Covid-19 case where treatment is frequently aimed at the symptoms rather than the actual disease.
